apps/storefront/components/CodeSnippet/CodeSnippet.tsx

```diff
@@ -1,49 +1,60 @@
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import { FilesIcon } from '@navikt/aksel-icons';
 import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter';
-import prettier from 'prettier/standalone.js';
-import parserJs from 'prettier/parser-flow.js';
-import parserHtml from 'prettier/parser-markdown.js';
-import parserCss from 'prettier/parser-postcss.js';
-import parserTs from 'prettier/parser-typescript';
+import type { Options } from 'prettier';
```

Comment:
```suggestion
```

I think you forgot to remove this import

Reply:
Fixed